src/main/webapp/index.html

```diff
@@ -30,5 +32,7 @@
       <input type="text" placeholder="Longitude" id="longitude" readonly />
     </div>
     <button type="button" id="my-location">Go to my location</button>
+    <button type="button" id="videos">Videos</button>
+    <script src="https://apis.google.com/js/client.js?onload=handleClientLoad"></script>
```

Comment:
Does this need to be down here?
If not, I think it would be better if it goes around lines 7-13 with the other scripts.

Also, I think you can call the onload inside of the js file if you need